bc/core/utils/images.py

```diff
@@ -0,0 +1,273 @@
+import io
+from dataclasses import dataclass, field
+from math import ceil, sqrt
+from pathlib import Path
+from textwrap import fill, wrap
+
+from PIL import Image, ImageFont
+from PIL.ImageDraw import Draw
+
+from bc.settings import STATIC_GLOBAL_ROOT
+
+
+@dataclass
+class TextImage:
+    title: str
+    description: str
+    title_font_path: Path = (
+        STATIC_GLOBAL_ROOT / "fonts" / "CooperHewitt-Bold.otf"
+    )
+    desc_font_path: Path = (
+        STATIC_GLOBAL_ROOT / "fonts" / "CooperHewitt-Light.otf"
+    )
+    font_size: int = 24
+    font_spacing: int = 16
```

Comment:
Font spacing would normally mean the distance between letters, also known as the kerning. What you mean here is line height, right? If so:

```suggestion
    line_height: int = 16
```

Or perhaps:

```suggestion
    line_spacing: int = 16
```

Reply:
You're right about the `font_spacing`.

I like your second suggestion because We're using this `attribute` to control the `spacing` parameter of the [multiline_text](https://pillow.readthedocs.io/en/stable/reference/ImageDraw.html#PIL.ImageDraw.ImageDraw.multiline_text) method from the [ImageDraw ](https://pillow.readthedocs.io/en/stable/reference/ImageDraw.html) module and they use the following description in the docs: 

> spacing – The number of pixels between lines.

It's really vague but I think they mean that this parameter sets **the vertical gap** between lines.

I don't think We should call it `line_height` because We're already using that name to represent the combined measurement of the `font_size` and this space between lines. 

I thought about calling this attribute **leading** but It seemed too technical.